empress/recon_vis/tree.py

```diff
@@ -19,17 +24,52 @@ def __init__(self, name):
         self.parent_node = None     # Node:  parent Node or None
         self.layout = None          # NodeLayout object: layout of this node
 
+    # The @property decorator allows this to be called as .is_leaf rather than .is_leaf()
+    @property
```

Comment:
Please remove this decorator and its corresponding comments.

Reply:
That is, use `node.is_leaf()` instead of `node.is_leaf`.

Reply:
Fixed